templates/vpc.yaml

```diff
@@ -51,8 +51,18 @@ Parameters:
       - true
       - false
     Default: true
+  IncludeBastianSecurityGroup:
+    Type: String
+    Description: >
+      true (default) to deploy a bastian security group
+      false to skip the bastian security group
+    AllowedValues:
+      - true
+      - false
+    Default: true
```

Comment:
I don't see how this fixes issue IT-2000.  This default is telling CFN to create this SG unless it's explicitly set to `false`.  Do you intend to set this to false on every VPC deployment?

Reply:
I thought about setting false as default but that would have removed the sg everywhere (my concern was that some account that uses it would suddenly lose it). So I put true to keep things as is, then we explicitly specify true where we needed, then set default as false to remove (too contrived?).